pyanaconda/modules/storage/partitioning/manual_partitioning.py

```diff
@@ -34,19 +32,25 @@
 class ManualPartitioningTask(NonInteractivePartitioningTask):
     """A task for the manual partitioning configuration."""
 
+    def __init__(self, storage, requests):
+        """Create a task.
+
+        :param storage: an instance of Blivet
+        :param requests: a list of requests
+        """
+        super().__init__(storage)
+        self._requests = list(requests)
+
     def _configure_partitioning(self, storage):
         """Configure the partitioning.
 
         :param storage: an instance of Blivet
         """
         log.debug("Setting up the mount points.")
-        manual_part_proxy = STORAGE.get_proxy(MANUAL_PARTITIONING)
-
-        # Set up mount points.
-        for mount_data in manual_part_proxy.Requests:
-            self._setup_mount_point(storage, MountPointRequest.from_structure(mount_data))
+        for mount_data in self._requests:
+            self._setup_mount_point(storage, mount_data)
 
-    def _setup_mount_point(self, storage, mount_data):
+    def _setup_mount_point(self, storage, mount_data: MountPointRequest):
```

Comment:
It's already in the documentation, what is the reason to specify it here for a protected method?

Reply:
Fixed.